docs/ops.html

```diff
@@ -1271,6 +1271,24 @@ <h4 class="anchor-heading"><a id="remote_jmx" class="anchor-link"></a><a href="#
         <td>kafka.server:type=BrokerTopicMetrics,name=ReplicationBytesInPerSec</td>
         <td></td>
       </tr>
+      <tr>
+        <td>Controller Request rate from Broker</td>
+        <td>kafka.controller:type=ControllerChannelManager,name=RequestRateAndQueueTimeMs,brokerId=([-.\w]+)</td>
```

Comment:
Our brokerId should be an Integer value, except I miss anything.
ref: http://kafka.apache.org/documentation/#brokerconfigs_broker.id

Reply:
@showuon You are right. Fixed.